react/what-is-props/what-is-props.md

```diff
@@ -0,0 +1,131 @@
+# What is props?
+
+---
+
+If you are beginner in react you must have heard about `props` a lot, so let's understand, how is it so important to understand.
+
+## How is it useful?
+
+Before starting we need to understand how react works. So React is a component based `JavaScript` Library where we can divide our project into various reusable components.
+
+For example the below code is your main `App.js` file which you can see when we create a project using create-react-app
```

Comment:
Say `below code will be present in your main when you create a project`, instead of otherwise.

Why? Fewer words, more meaning.